src/main/java/com/epam/tc/api/hw3/dto/BoardDTO.java

```diff
@@ -0,0 +1,20 @@
+package com.epam.tc.api.hw3.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.Accessors;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Accessors(chain = true)
```

Comment:
It looks like these annotations are not needed, as @Data includes getters and RequiredArgsConstructor and @Accessors is not used anywhere